### PR TITLE
Style and performance improvements

### PR DESCRIPTION
**Pull request by Gauravjeetsingh**
Use fontawesome icon fonts for next and previous button
Add a loader when changing the ang
Add custom gesture; double tap to enable/disable larivaar assist
Implement caching of API requests

### CODE REVIEW

**Comment by theamanjs** [comment]
@Gauravjeetsingh veerji, please resolve conflicts in this PR

File: components/Ang/utils/sendRequest.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        setCurrentAngData(newData);
+        setIsLoading(false);
+      });
+      getFromAPI(page + 1);
+    }
+  });
+};
+
+const getFromAPI = async (page: number) => {
+  const dataRequest = await fetch(`https://api.banidb.com/v2/angs/${page}`);
```
Comment (theamanjs) [inline components/Ang/utils/sendRequest.tsx]: Recommended to put the base url of APIs in a constant variable

File: components/common/double-tap.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import React, {useRef} from 'react';
+import {State, TapGestureHandler} from 'react-native-gesture-handler';
+
+export const DoubleTap = ({children, customTap}: any) => {
+  const doubleTapRef = useRef(null);
+
+  const onDoubleTapEvent = (event: any) => {
+    if (event.nativeEvent.state === State.ACTIVE) {
+      console.log(customTap);
```
Comment (theamanjs) [inline components/common/double-tap.tsx]: Do we need this console.log for any reason?

**Review by theamanjs — CHANGES_REQUESTED** [review]

File: components/Ang/utils/sendRequest.tsx (hunk 1 above)
Comment (Gauravjeetsingh) [reply components/Ang/utils/sendRequest.tsx]: Done veerji

File: components/common/double-tap.tsx (hunk 2 above)
Comment (Gauravjeetsingh) [reply components/common/double-tap.tsx]: No, we don't. Removed it.

**Pull request merged**